app/views/govuk_publishing_components/components/_document_list.html.erb

```diff
@@ -45,5 +49,7 @@
         <% end %>
       </li>
     <% end %>
-  </ol>
+  <% unless within_multitype_list %>
```

Comment:
I'm not sure how this would work, can you please give an example? 🙂

Reply:
I was thinking you'd wrap up the code in a block somehow and then do an if statement of some kind of either just output the block or do `content_tag('ol', block)`. Thinking about this in more detail it's probably not appropriate for this situation, as you either want the wrapping ol or you don't - this would work better where you would want a choice of wrapping elements.